worker/worker_doc_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//nolint:all // This is sample code
+package worker_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/abcxyz/pkg/worker"
+)
+
+func Example_sleep() {
+	ctx := context.TODO()
+	w := worker.New[*worker.Void](3)
+
+	for i := 0; i < 5; i++ {
+		if err := w.Do(ctx, func() (*worker.Void, error) {
+			time.Sleep(10 * time.Millisecond)
+			return nil, nil
+		}); err != nil {
+			// TODO: check err
+		}
+	}
+
+	results, err := w.Done(ctx)
+	if err != nil {
+		// TODO: check err
+	}
+	_ = results
+}
+
+func Example_hTTP() {
```

Comment:
hTTP -> HTTP

Reply:
I agree this is super annoying, but it's actually intentional. It can't be `HTTP` because it has to start with a lowercase letter. If I make it `_http`, then the example name looks silly:

<img width="160" alt="screenshot-20221007-rgQkooBa@2x" src="https://user-images.githubusercontent.com/408570/194669520-a6daa78d-d298-470c-98fb-d8f38104e20c.png">

Here's how it looks as-written:

<img width="216" alt="screenshot-20221007-D3nMJ2C7@2x" src="https://user-images.githubusercontent.com/408570/194669532-ea571415-366b-4ac0-964b-974163e9d45e.png">